src/util.cpp

```diff
@@ -478,7 +478,15 @@ const boost::filesystem::path &ZC_GetParamsDir()
     if (!path.empty())
         return path;
 
-    path = ZC_GetBaseParamsDir();
+    if (mapArgs.count("-paramsdir")) {
```

Comment:
This code should be in ``ZC_GetBaseParamsDir``; ``ZC_GetParamsDir`` is only supposed to return the cached value.

Reply:
ok